tests/driver_ws281x/Makefile

```diff
@@ -1,4 +1,4 @@
-BOARD ?= atmega328p
+BOARD ?= atmega1284p
```

Comment:
Don't know how this slipped in - I'll change it back.

Reply:
You could have kept this ;-) There was absolutely no reason for having the `atmega328p` "board" as default other than it has (or is :-D) an ATmega MCU, and so is the ATmega1284P :-)